xchange-okex/src/main/java/org/knowm/xchange/okex/v5/OkexResilience.java

```diff
@@ -0,0 +1,45 @@
+package org.knowm.xchange.okex.v5;
+
+import io.github.resilience4j.ratelimiter.RateLimiterConfig;
+import org.knowm.xchange.client.ResilienceRegistries;
+import org.knowm.xchange.client.ResilienceUtils;
+
+import java.time.Duration;
+
+import static javax.ws.rs.core.Response.Status.TOO_MANY_REQUESTS;
+
+/** Author: Max Gao (gaamox@tutanota.com) Created: 08-06-2021 */
+public class OkexResilience {
+
+  public static final String PUBLIC_REST_ENDPOINT_RATE_LIMITER = "publicEndpointLimit";
+
+  public static final String PRIVATE_REST_ENDPOINT_RATE_LIMITER = "privateEndpointLimit";
+
+  public static ResilienceRegistries createRegistries() {
+    final ResilienceRegistries registries = new ResilienceRegistries();
+
+    registries
+        .rateLimiters()
+        .rateLimiter(
+            PUBLIC_REST_ENDPOINT_RATE_LIMITER,
+            RateLimiterConfig.from(registries.rateLimiters().getDefaultConfig())
+                .limitRefreshPeriod(Duration.ofSeconds(1))
+                .limitForPeriod(3)
```

Comment:
are these rate limits based on OKex limits?

Reply:
Will be updating these in a later commit to this PR. 

Each endpoint will have its own rate limit. I'm in the process of coming up with something to handle this gracefully.

Reply:
Made a change and ran a quick test by submitting exchange.remoteInit() to an executor for asynchronous execution.

I might have misconfigured something, but on first glance it doesn't seem like the rate limiter is thread safe?

```12:05:27.161 [main] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.171 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.171 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.174 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.174 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.177 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.405 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:27.429 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.439 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.451 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:27.453 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:27.455 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:27.464 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:27.490 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.492 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.505 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.505 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.507 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.508 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.508 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.510 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.680 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:27.691 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.694 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.727 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:27.738 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:27.743 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.745 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.746 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:27.760 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:27.768 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.770 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.777 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.778 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.780 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.782 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.943 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:27.953 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.956 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:27.986 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:27.998 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:27.999 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.025 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:28.026 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:28.033 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:28.054 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:28.056 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.057 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:28.064 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:28.065 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.066 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.204 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:28.223 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:28.229 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.239 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:28.250 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:28.252 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.290 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 200
12:05:28.303 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 429
12:05:28.303 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 429
12:05:28.304 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 429; response body:
{"msg":"Requests too frequent.","code":50011}
12:05:28.304 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 429; response body:
{"msg":"Requests too frequent.","code":50011}
12:05:28.305 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 200; response body:
{"code":"0","data":[{"alias":"","baseCcy":"BTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"BTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.00000001","minSz":"0.00001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.1","uly":""},{"alias":"","baseCcy":"ETH","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETH-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.001","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"OKB","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKB-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"OKT","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"OKT-USDT","instType":"SPOT","lever":"","listTime":"1611012701000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"LTC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"LTC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.01","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"DOT","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOT-USDT","instType":"SPOT","lever":"5","listTime":"1598061640000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"DOGE","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"DOGE-USDT","instType":"SPOT","lever":"5","listTime":"1548133413000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.000001","uly":""},{"alias":"","baseCcy":"XRP","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"XRP-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00001","uly":""},{"alias":"","baseCcy":"EOS","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"EOS-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"ETC","category":"1","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"ETC-USDT","instType":"SPOT","lever":"10","listTime":"1548133413000","lotSz":"0.000001","minSz":"0.1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.001","uly":""},{"alias":"","baseCcy":"MATIC","category":"2","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MATIC-USDT","instType":"SPOT","lever":"5","listTime":"1617008400000","lotSz":"0.000001","minSz":"10","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.0001","uly":""},{"alias":"","baseCcy":"MINA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"MINA-USDT","instType":"SPOT","lever":"5","listTime":"1622508605000","lotSz":"0.000001","minSz":"1","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.01","uly":""},{"alias":"","baseCcy":"AKITA","category":"3","ctMult":"","ctType":"","ctVal":"","ctValCcy":"","expTime":"","instId":"AKITA-USDT","instType":"SPOT","lever":"5","listTime":"1620979264000","lotSz":"1","minSz":"1000000","optType":"","quoteCcy":"USDT","settleCcy":"","state":"live","stk":"","tickSz":"0.00000001","uly"
12:05:28.312 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
org.knowm.xchange.okex.v5.dto.OkexException: Requests too frequent.
org.knowm.xchange.okex.v5.dto.OkexException: Requests too frequent.
12:05:28.318 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.318 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.472 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 429
12:05:28.472 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 429; response body:
{"msg":"Requests too frequent.","code":50011}
org.knowm.xchange.okex.v5.dto.OkexException: Requests too frequent.
12:05:28.473 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.489 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 429
12:05:28.489 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 429; response body:
{"msg":"Requests too frequent.","code":50011}
org.knowm.xchange.okex.v5.dto.OkexException: Requests too frequent.
12:05:28.489 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.548 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 429
12:05:28.548 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 429; response body:
{"msg":"Requests too frequent.","code":50011}
org.knowm.xchange.okex.v5.dto.OkexException: Requests too frequent.
12:05:28.549 [pool-1-thread-3] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.549 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 429
12:05:28.550 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 429; response body:
{"msg":"Requests too frequent.","code":50011}
org.knowm.xchange.okex.v5.dto.OkexException: Requests too frequent.
12:05:28.550 [pool-1-thread-2] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.554 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 429
12:05:28.554 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 429; response body:
{"msg":"Requests too frequent.","code":50011}
org.knowm.xchange.okex.v5.dto.OkexException: Requests too frequent.
12:05:28.555 [pool-1-thread-4] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.723 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 429
12:05:28.724 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 429; response body:
{"msg":"Requests too frequent.","code":50011}
org.knowm.xchange.okex.v5.dto.OkexException: Requests too frequent.
12:05:28.727 [pool-1-thread-5] DEBUG si.mazi.rescu.HttpTemplate - Executing GET request at https://www.okex.com/api/v5/public/instruments?instType=SPOT
12:05:28.742 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Request http status = 429
12:05:28.742 [pool-1-thread-1] DEBUG si.mazi.rescu.HttpTemplate - Http call returned 429; response body:
{"msg":"Requests too frequent.","code":50011}
org.knowm.xchange.okex.v5.dto.OkexException: Requests too frequent.```

Reply:
Pretty sure it is, did you enable the rate limiter when you started it?

Reply:
Does it need to be explicitly enabled somewhere in addition to creating the registries and passing them into services + decorating the calls from that point onward?

Reply:
nah its an exchange spec

Reply:
Enabled and tested. 
It seems that effective rate limits need to be lowered from the values supplied in exchange APIs to avoid 429s completely.